src/test/java/ca/qaguru/oranghrmbatch27/tests/EmploymentStatusTests.java

```diff
@@ -0,0 +1,19 @@
+package ca.qaguru.oranghrmbatch27.tests;
+
+import ca.qaguru.oranghrmbatch27.library.TestBase;
+import ca.qaguru.oranghrmbatch27.pages.*;
+import org.testng.annotations.Test;
+
+public class EmploymentStatusTests extends TestBase {
+
+
+    @Test
+    public void addNewEmploymentStatus() {
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.login("admin", "admin123", true, null);
+        HeaderPage headerPage = new HeaderPage(driver);
+        headerPage.selectMenu(MenuOptions.EMPLOYMENT_STATUS);
+        EmploymentStatusPage employmentStatusPage = new EmploymentStatusPage(driver);
+        employmentStatusPage.addEmploymentStatus("Freelance2");
```

Comment:
String uuid = UUID.randomUUID().toString();
employmentStatusPage.addEmploymentStatus("Freelance" + uuid);

rename your method addEmploymentStatus to addNewEmploymentAndVerifyIsDisplayed